data/darktableconfig.xml.in

```diff
@@ -15,6 +15,27 @@
     <shortdescription>database location</shortdescription>
     <longdescription>filename relative to ~/.config/darktable or starting with a slash (needs a restart).</longdescription>
   </dtconfig>
+  <dtconfig>
```

Comment:
I would add this chunk into the preference dialog, no?

You need to specify attribute `prefs="core"` in dtconfig node.

Reply:
Well I'll be damned, doesn't look half-bad there!